# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/11
# install relion3 wrappers into bin
diff -ru topaz-0.2.5/setup.py topaz-0.2.5_install_relion3_wrappers/setup.py
--- topaz-0.2.5/setup.py	2022-03-11 02:33:01.000000000 +0100
+++ topaz-0.2.5_install_relion3_wrappers/setup.py	2023-11-13 17:46:28.306807471 +0100
@@ -30,7 +30,7 @@
     #package_dir = {'': 'topaz'},
     entry_points = {'console_scripts': ['topaz = topaz.main:main']},
     include_package_data = True,
-
+    scripts=['relion_run_topaz/run_topaz_%s.py' % x for x in ['denoise','pick','train','train_denoise']],
     install_requires=[
         'torch>=1.0.0',
         'torchvision',
